Mathlib/CategoryTheory/Limits/Shapes/NormalMono/Equalizers.lean

```diff
@@ -37,14 +37,12 @@ def pullback_of_mono {X Y Z : C} (a : X ⟶ Z) (b : Y ⟶ Z) [Mono a] [Mono b] :
         _ = kernel.ι (prod.lift f g) ≫ prod.lift f g ≫ Limits.prod.fst := by rw [prod.lift_fst]
         _ = (0 : kernel (prod.lift f g) ⟶ P ⨯ Q) ≫ Limits.prod.fst := by rw [kernel.condition_assoc]
         _ = 0 := zero_comp
-
```

Comment:
Arguably, this proof could use some comments - but adding those is not the job of this PR.